src/System.Runtime.Extensions/tests/System/Environment.Exit.cs

```diff
@@ -42,6 +42,15 @@ public static void CheckExitCode(int expectedExitCode)
 
         [Theory]
         [MemberData(nameof(ExitCodeValues))]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.UapNotUapAot, "RemoteExecutor behaves different in UAP.")]
```

Comment:
I'm not understanding the code snippet. Can you elaborate on how that if check in the body of the using would help?